ActiveRecord.php

```diff
@@ -415,7 +415,7 @@ public static function instantiate($row)
      * By default the `op_type` is set to `create`.
      * @return boolean whether the attributes are valid and the record is inserted successfully.
      */
-    public function insert($runValidation = true, $attributes = null, $options = ['op_type' => 'create'])
+    public function insert($runValidation = true, $attributes = null)
```

Comment:
options could be used to define routing for parent child relations before. Is there a different way to do this now?